tests/unit/api/opHandlers/OpHandlerResponseDateRangeConditionTest.php

```diff
@@ -31,7 +31,13 @@ public function testExecuteMinAndMaxBuildsBetweenConditionWithNormalizedTimes():
         $criteria = $handler->execute('created_at', ['2024-07-01', '2024-07-31']);
 
         $this->assertInstanceOf(\CDbCriteria::class, $criteria);
-        $this->assertSame('`created_at` BETWEEN :created_atMin AND :created_atMax', $criteria->condition);
+        //$this->assertSame('`created_at` BETWEEN :created_atMin AND :created_atMax', $criteria->condition);
+        $this->assertTrue(
```

Comment:
I would recommend the implementation of helper functions here too, like described at tests/unit/api/opHandlers/OpHandlerResponseContainConditionTest.php of course, making them as generic as possible and hence not repeating the same logic three times one below the other to handle dialects, to avoid code bloating and handling the dialects once per logic type. Of course, this stands for all subsequent code repetitions in this file due to dialect support.